pkg/server/transaction_diagnostics_requests.go

```diff
@@ -245,3 +253,114 @@ func (s *statusServer) TransactionDiagnosticsRequests(
 	}
 	return response, nil
 }
+
+// getTransactionBundle retrieves a transaction diagnostic bundle by ID and
+// writes it out as an attachment. This includes the transaction bundle itself
+// and all associated statement bundles ordered by their ID. Note this function
+// assumes the user has permission to access the transaction bundle.
+func (s *adminServer) getTransactionBundle(ctx context.Context, id int64, w http.ResponseWriter) {
```

Comment:
Should we just inline this function into `TxnBundleHandler`

Reply:
yes! done.